client/src/components/Card/Rate.js

```diff
@@ -28,16 +28,30 @@ const YellowBall = styled(Ball)`
   height: 1.5rem;
 `;
 
-export default function Rate() {
+export default function Rate({ rate }) {
+  const balls = [];
+
+  function Rating(rate) {
+    const intRate = parseInt(rate);
+    console.log(intRate);
```

Comment:
I'm not sure but are the console.logs necessary?

Reply:
Not really :) . I removed all console.logs. Thank you!